src/main/java/com/stripe/model/Topup.java

```diff
@@ -0,0 +1,213 @@
+package com.stripe.model;
+
+import com.stripe.exception.*;
+import com.stripe.net.APIResource;
+import com.stripe.net.RequestOptions;
+
+import java.util.Map;
+
+public class Topup extends APIResource implements MetadataStore<Topup>, HasId {
+    String id;
+    String object;
+    Integer amount;
+    ExpandableField<BalanceTransaction> balanceTransaction;
+    Long created;
+    String currency;
+    String description;
+    Long expectedAvailabilityDate;
+    String failureCode;
+    String failureMessage;
+    Map metadata;
```

Comment:
Nit: Noticed you dropped the type parameters here.  Intended?

Reply:
Oops, my bad. Fixed in 1b51c2c